packages/render/src/primitives/form/renderTextInput.js

```diff
@@ -0,0 +1,29 @@
+import { parseTextFieldOptions } from '../../utils/parseFormOptions';
+
+const renderTextInput = (ctx, node, options) => {
+  const { top, left, width, height } = node.box || {};
+
+  // Element's name
+  const name = node.props?.name || '';
+  const formFieldOptions = options.formFields?.at(0);
+
+  if (!formFieldOptions)
```

Comment:
Checkboxes and TextFields don't need a FormField to them. Would be recommended but not necessary.